src/services/__tests__/authService.test.js

```diff
@@ -0,0 +1,31 @@
+import { apiUrl } from '../../config.json';
+import httpService from '../httpService';
+import * as authService from '../authService';
+
+jest.mock('../httpService.js', () => {
+  return {
+    get: jest.fn(),
+    post: jest.fn()
+  };
+});
+
+const email = 'abc';
```

Comment:
can we make these constants all caps and delimited by underscores? URL_ENDPOINT for example